app/controllers/api/skater_profiles_controller.rb

```diff
@@ -0,0 +1,32 @@
+class Api::SkaterProfilesController < ApplicationController
+  def show
+    profile = SkaterProfile.find_by(skater_id: current_skater.id)
+
+    render json: profile
+  end
+
+  def create
+    profile = SkaterProfile.new(create_params)
+    profile.save!
+
+    render json: profile
+  end
+
+  def update
+    profile = SkaterProfile.find_by(skater_id: current_skater.id)
+    profile.attributes = update_params
+    profile.save!
+
+    render json: profile
+  end
+
+  private
+
+  def create_params
```

Comment:
Since create_params and update_params are the same, we could just have one method called profile_params (not that big of a deal since it will likely need to be different in the future maybe?)